tests/integration_tests.rs

```diff
@@ -2,11 +2,11 @@ use async_memcached::{Client, Error, ErrorKind, Status};
 use rand::seq::IteratorRandom;
 use serial_test::{parallel, serial};
 
-// Note: Each test should run with keys unique to that test to avoid async conflicts.  Because these tests run concurrently,
+// NOTE: Each test should run with keys unique to that test to avoid async conflicts.  Because these tests run concurrently,
 // it's possible to delete/overwrite keys created by another test before they're read.
 
-const LARGE_PAYLOAD_SIZE: usize = 1000 * 1024;
-const MAX_KEY_LENGTH: usize = 250;
+const LARGE_PAYLOAD_SIZE: usize = 1000 * 1024; // 1 MB, default memcached max value size
```

Comment:
It's technically 1MiB (1024*1024)

Reply:
The entire request has to be <1MiB so we use 1024*1024 - 250b (maximum key length) - 60b (metadata).